core/src/test/java/org/apache/druid/java/util/http/client/FriendlyServersTest.java

```diff
@@ -275,6 +276,8 @@ public void testFriendlySelfSignedHttpsServer() throws Exception
 
     sslConnector.setPort(0);
     server.setConnectors(new Connector[]{sslConnector});
+    KeyStoreScanner keyStoreScanner = new KeyStoreScanner(sslContextFactory);
```

Comment:
This wouldn't detect keystore changes if the keystore path was a soft link, would it?

Reply:
Good question, https://github.com/eclipse/jetty.project/blob/jetty-10.0.x/tests/test-integration/src/test/java/org/eclipse/jetty/test/KeyStoreScannerTest.java#L182-L206 leads me to believe that it has that ability.